tests/integration/components/cf-field-test.js

```diff
@@ -46,6 +46,16 @@ module("Integration | Component | cf-field", function(hooks) {
     this.set("field", document.fields[0]);
   });
 
+  test("it allows deleting existing input", async function(assert) {
+    assert.expect(1);
+
+    await render(hbs`{{cf-field field=field}}`);
+    this.set("field.answer.value", "Test");
+
+    await fillIn("input", "");
+    assert.equal(this.field.answer.value, "", "Value was removed.");
```

Comment:
Shouldn't the whole answer be removed?

Reply:
Good point! I'll look into that. Probably need to add Mirage to the test.

Reply:
I don't really see a way to create the fields from (Mirage)GraphQL without the cf-form component. I'll have another look at it later. This is probably going to take a moment.

Reply:
Wanna create an issue and move on for the moment?

Reply:
Maybe the best solution to get stuff done quickly.

Reply:
Alright, please don't forget to create the issue :wink:

Reply:
https://github.com/projectcaluma/ember-caluma/issues/181